spec/test-outputs/www-integration.out.vcl

```diff
@@ -510,5 +518,7 @@ sub vcl_pass {
 }
 
 sub vcl_hash {
+  # support caching non-GET/HEAD requests
+  set req.hash += req.request;
```

Comment:
This will ensure that we cache GET and POST request responses uniquely, so when making a POST request you won't receive the cached response to a GET request.